tests/TileParserTests.cs

```diff
@@ -21,7 +20,7 @@ public void TestIssue10MapBoxVectorTile()
         const string mapboxissue10File = "mapbox.vector.tile.tests.testdata.cadastral.pbf";
 
         // act
-        var pbfStream = Assembly.GetExecutingAssembly().GetManifestResourceStream(mapboxissue10File);
+        var pbfStream = TestData.GetTestResource(mapboxissue10File);
```

Comment:
For the tests I prefer to use FIle.OpenRead, so TestData.GetTestResource is not needed.

 Like: 

```
        const string bagfile = "testdata/bag-17-67317-43082.pbf";
        var pbfStream = File.OpenRead(bagfile);
```

And testfile settings to:

![image](https://github.com/user-attachments/assets/aa596991-37df-4d2f-8b61-2388531167f1)